fc/cli/watch.py

```diff
@@ -0,0 +1,27 @@
+import click
+from ..jira.issue import Issue
+from ..auth.combo import ComboAuth
+from . import cli_library
+
+
+@click.command()
+@click.option('--username')
+@click.argument('issue_ids', nargs=-1, metavar='ISSUE-1 ISSUE-2 ISSUE-3 etc.')
+def watch(username: str, issue_ids: tuple):
```

Comment:
This really should be a command under the existing `fcli task`.  So `fcli task watch`.

Reply:
If you'd prefer, I make that happen.

Reply:
agreed

Reply:
OK so I moved and it seems to work.